Packs/FortiManager/Integrations/FortiManager/FortiManager_description.md

```diff
@@ -0,0 +1,4 @@
+## FortiManager
+- FortiManager is a single console central management system that manages Fortinet Devices.
+- You can use FortiManger integration to:
+  - View, create, update and delete Addresses, Address-Groups, Services, Service-Groups, Policies and Policy-Packages found in FortiManger.
```

Comment:
maybe add an explanation for getting the creds

Reply:
Not sure it is necessary, it's just the username and password for FortiManager.